.github/workflows/periodic-cleanup.yaml

```
@@ -13,33 +22,61 @@ jobs:
+          OC_VERSION="stable"
+          curl -sL \
+            https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux.tar.gz \
+            -o oc.tar.gz
+          tar -xzf oc.tar.gz oc
+          chmod +x oc
+          sudo mv oc /usr/local/bin/
+          oc version --client
+
+      # Use the oc cmds with sa token to delete resources
+      - name: Cleanup Konflux staging resources
+        id: konflux
```

actionlint (1.7.11)

[error] step ID "konflux" duplicates. previously defined at line:58,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)